mockstore/mocktikv/errors_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2021 TiKV Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// NOTE: The code in this file is based on code from the
+// TiDB project, licensed under the Apache License v 2.0
+//
+// https://github.com/pingcap/tidb/tree/cc5e161ac06827589c4966674597c137cc9e809c/store/tikv/mockstore/mocktikv/mock_tikv_test.go
+//
+
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mocktikv
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestErrors(t *testing.T) {
+	assert := assert.New(t)
+	assert.Equal((&ErrKeyAlreadyExist{}).Error(), `key already exist, key: ""`)
+	assert.Equal(ErrAbort("txn").Error(), "abort: txn")
+	assert.Equal(ErrAlreadyCommitted(0).Error(), "txn already committed")
+	assert.Equal((&ErrConflict{}).Error(), "write conflict")
```

Comment:
Why do we test this things...It seems like we test getter/setter.

Does the error string should keep compatible?

Reply:
Good point. I did some code search, at least in TiDB there is no usage rely on the specific strings. I think we'd better to remove this test.